src/Aspire.Dashboard/Components/Controls/ResourceDetails.razor

```diff
@@ -313,21 +313,21 @@
 
     private static RenderFragment RenderTextValue(DisplayedUrl vm, string filter)
     {
-        var highlighting = !string.IsNullOrEmpty(filter) && vm.Text != "-";
+        var highlighting = !string.IsNullOrEmpty(filter) && !string.IsNullOrEmpty(vm.DisplayName);
 
-        // If there's no URL, e.g. this is a tcp:// URI, show nothing, or URL is same as Text, then show nothing
-        if (vm.Url is null || string.Equals(vm.Url, vm.Text, StringComparison.Ordinal))
+        // If there's no DisplayName then show nothing
+        if (string.IsNullOrEmpty(vm.DisplayName))
         {
-            return @<span></span>;
+            return @<span>-</span>;
```

Comment:
In other places we have `<span class="empty-data"></span>` and the dash is added by CSS. Change this to be consistent.

https://github.com/dotnet/aspire/blob/3f9e7d28f9ce32e0c4d0036409a7fe2a27ac6ae6/src/Aspire.Dashboard/Components/Controls/GridValue.razor#L21